_includes/v20.1/orchestration/start-kubernetes.md

```diff
@@ -11,20 +11,24 @@ Choose whether you want to orchestrate CockroachDB with Kubernetes using the hos
 
     This includes installing `gcloud`, which is used to create and delete Kubernetes Engine clusters, and `kubectl`, which is the command-line tool used to manage Kubernetes from your workstation.
 
+    {{site.data.alerts.callout_success}}
+    Be sure to set a [default compute zone and/or region](https://cloud.google.com/kubernetes-engine/docs/quickstart#defaults) to use with `gcloud`. If no defaults are set, you will need to specify them with the `--zone` and `--region` flags when creating and deleting clusters.
+    {{site.data.alerts.end}}
+
     {{site.data.alerts.callout_success}}The documentation offers the choice of using Google's Cloud Shell product or using a local shell on your machine. Choose to use a local shell if you want to be able to view the CockroachDB Admin UI using the steps in this guide.{{site.data.alerts.end}}
 
 2. From your local workstation, start the Kubernetes cluster:
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ gcloud container clusters create cockroachdb --machine-type n1-standard-4
+    $ gcloud container clusters create cockroachdb --machine-type n2-standard-4
```

Comment:
This creates a zonal cluster and not a regional one, we should maybe note that `regional` is recommended for production use.